apps/prairielearn/src/pages/editError/editError.html.ts

```diff
@@ -22,6 +23,7 @@ export function EditError({
     <html lang="en">
       <head>
         ${HeadContents({ resLocals, pageTitle: 'Edit Failure' })}
+        ${compiledScriptTag('editErrorClient.ts')}
```

Comment:
To test this change: add something like this to line 290 in editors.ts:

```ts
      job.verbose('Testing a verbose message.');
      throw new Error('Testing edit_error page.');
```

Then perform an action that relies on the editors, like adding an assessment.